docker/Dockerfile-centos7-builder

```diff
@@ -0,0 +1,25 @@
+FROM centos:7
+
+ENV LANG en_US.utf8
+WORKDIR /greptimedb
+
+RUN sed -e 's|^mirrorlist=|#mirrorlist=|g' \
+             -e 's|^#baseurl=http://mirror.centos.org/centos|baseurl=http://mirrors.tuna.tsinghua.edu.cn/centos|g' \
+             -i.bak \
+             /etc/yum.repos.d/CentOS-*.repo
+
+# Install dependencies.
+RUN ulimit -n 1024000 && yum install -y epel-release openssl openssl-devel centos-release-scl
```

Comment:
I was passing by and decided to put my two cents.
Maybe it makes sense to squash `yum install` into a single instruction. If I recall correctly each `RUN` creates a layer thus increasing the image size.
Alternatively (obviously more work) you might consider using the builder pattern.

Reply:
> I was passing by and decided to put my two cents. Maybe it makes sense to squash `yum install` into a single instruction. If I recall correctly each `RUN` creates a layer thus increasing the image size. Alternatively (obviously more work) you might consider using the builder pattern.

Sure. I'll squash all `yum install` into one line.